### ISSUE TITLE
[Feature Request] Epub support

### ISSUE THREAD

**Issue by RickyM7** [max-kammerer/orion-viewer#29]
It would be great if the app supported reading epub files as well.

**Comment by max-kammerer** [comment]
Could you clarify your request. There are already a lot of application for Android that supports epub format

**Comment by RickyM7** [comment]
> There are already a lot of application for Android that supports epub format

Yes, you're right, however these apps almost always only only serve exclusively for this or they are very heavy, however Orion Viewer is relatively very light and has a good usability and support the epub format would be a great addition to the app in my opinion.

**Comment by seniorm0ment** [comment]
This is something I would like to see as well.

**Comment by RickyM7** [comment]
> This is OT, but @RickyM7 , I'd appreciate it if you could you point me to a Free app that is exclusively for .epub?

[This](https://f-droid.org/pt_BR/packages/com.quaap.bookymcbookface/), maybe? I've created this issue a lot of time ago, so i can't remember well which apps i was talking about, sorry.

**Comment by rimugu** [comment]
Epub (and maybe mobi) would be great.

**Comment by DiagonalArg** [comment]
> > This is OT, but @RickyM7 , I'd appreciate it if you could you point me to a Free app that is exclusively for .epub?
> 
> [This](https://f-droid.org/pt_BR/packages/com.quaap.bookymcbookface/), maybe? I've created this issue a lot of time ago, so i can't remember well which apps i was talking about, sorry.

Thanks, but I've tried it and it's buggy; and, it seems dead.  [project](https://github.com/quaap/BookyMcBookface)

**Comment by RickyM7** [comment]
> 
> Thanks, but I've tried it and it's buggy; and, it seems dead. [project](https://github.com/quaap/BookyMcBookface)

You're right. Have you take a look on [Librera](https://github.com/foobnix/LibreraReader)? I don't know if this project is or have been full open source any time, but it have epub support.

**Comment by SagXD** [comment]
@max-kammerer Any plan on adding this? MuPDF support epub and FB2. I don't want to install two readers to read books.
